python/samples/demos/telemetry/main.py

```diff
@@ -140,7 +140,7 @@ async def main(scenario: Literal["ai_service", "kernel_function", "auto_function
     with tracer.start_as_current_span("main") as current_span:
         print(f"Trace ID: {format_trace_id(current_span.get_span_context().trace_id)}")
 
-        stream = True
+        stream = False
```

Comment:
Is this change needed?

Reply:
Yes, because Azure AI Inference doesn't support streaming function calling yet.